.github/workflows/ci_build.yml

```diff
@@ -0,0 +1,64 @@
+name: Build production artifact
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+      - name: Cache node modules
+        uses: actions/cache@v4
+        with:
+          path: ./node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: Get Cypress version
+        id: cypress-version
+        run: echo "version=$(yarn info cypress version | tr -d '\n')" >> $GITHUB_OUTPUT
+
+      - name: Cache Cypress
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/Cypress
+          key: ${{ runner.os }}-cypress-${{ steps.cypress-version.outputs.version }}
+
+      - name: Cache Webpack
+        uses: actions/cache@v4
+        with:
+          path: node_modules/.cache
+          key: ${{ runner.os }}-webpack-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Cache Yarn
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/yarn
+          key: ${{ runner.os }}-yarn-cache-${{ hashFiles('**/yarn.lock') }}
+
+      - name: install dependencies
+        run: yarn install
+      - name: build production
+        run: yarn production-build
+      - name: Upload production build artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: prod
+          path: |
+            ./prod/dist
```

Comment:
You should be able to replace this with `./prod`. I think we always include everything in that folder.

Reply:
I did that intentionally, because it included the node_modules folder, making it an extra 750 MB to upload and then download. However, the node_modules are being cached, which is way faster than the artefact system.